frontend/apps/reader/modules/readerfont.lua

```diff
@@ -221,65 +218,64 @@ function ReaderFont:onSetFontSize(new_size)
     self.font_size = new_size
     self.ui.document:setFontSize(Screen:scaleBySize(new_size))
     self.ui:handleEvent(Event:new("UpdatePos"))
-    UIManager:show(Notification:new{
-        text = T( _("Font size set to %1."), self.font_size),
-    })
+    Notification:notify(T(_("Font size set to %1."), self.font_size), true)
```

Comment:
This `true` feels uneeded, as this notification is shown after the re-rendering has been done.